crates/telio-lana/src/event_log_file.rs

```diff
@@ -105,6 +105,20 @@ pub mod moose {
         pub config: MeshnetappContextApplicationConfig,
     }
 
+    /// Mock of moose::MeshnetappLogLevel.
+    #[derive(Debug, Clone, Deserialize, PartialEq, Serialize)]
+    /// value of 'log_level'
+    pub enum LibtelioappLogLevel {
```

Comment:
Might be cleaner to apply `#[serde(rename_all(serialize = "lowercase"))]` to the struct rather than manually renaming each variant

Reply:
Done